RELEASE.md

```diff
@@ -0,0 +1,47 @@
+# Release
+
+Here's how OAuth2_Proxy releases are created.
+
+## Schedule
+
+Our aim is to release once a quarter, but bug fixes will be prioritised and might be released earlier.
+
+## The Process
+
+Note this uses `v4.1.0` as an example release number.
+
+1. Create a draft Github release
+  * Use format `v4.1.0` for both the tag and title
+2. Update [CHANGELOG.md](CHANGELOG.md)
+  * Write the release highlights
+  * Copy in headings ready for the next release
+3. Create release commit
+  ```
+  git checkout -b release-v4.1.0
+  ```
+4. Create pull request getting other maintainers to review
+5. Copy the release notes in to the draft Github release, adding a link to the [changelog](CHANGELOG.md)
```

Comment:
The written `changelog` here is different to line 15, is that ok?

Reply:
It made sense at the time, but looking now I'll be consistent.